### DISCUSSION TITLE
Integration with external app

### DISCUSSION THREAD

**Discussion by interduo** [Etar-Group/Etar-Calendar#1499] [Q&A] [answered]
I am thinking about integrating with external app LMS (github:lmsgit/lms) and its timetable.

I could write an API in LMS providing methods:
- getEventList($startdate, $enddate),
- getEventDetails($id),
- changeEventState($id),
- changeEvent($params),
-...? 

Did somebody do integration like this before?
Is there any example?
What do You recommend?

**Comment by jspricke** [comment, marked as answer]
You probably want to use the Android calendar provider: https://developer.android.com/guide/topics/providers/calendar-provider
Etar is not the right abstraction level for that.
